keyboards/atset/at2/keyboard.json

```diff
@@ -0,0 +1,39 @@
+{
+    "manufacturer": "Atset",
+    "keyboard_name": "AT2",
+ 
+    "maintainer": "atset",
+    "board": "GENERIC_RP_RP2040",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    
+    
+    "diode_direction": "COL2ROW",
+    "usb": {
+        "vid": "0x2E8A",
```

Comment:
This vendor ID belongs to Raspberry Pi foundation, and the product ID looks to be assigned to the RP2 Boot device.   Please change the vendor ID.  Either 0xFEED, or something above 0x4000.

https://www.the-sz.com/products/usbid/index.php?v=0x2E8A&p=0x0003&n=